concordia/static/js/asset-reservation.js

```diff
@@ -1,7 +1,12 @@
 /* global jQuery displayMessage displayHtmlMessage buildErrorMessage */
 /* exported attemptToReserveAsset */
 
-function attemptToReserveAsset(reservationURL, findANewPageURL, actionType) {
+function attemptToReserveAsset(
+    reservationURL,
+    findANewPageURL,
+    actionType,
+    firstTime
```

Comment:
I'm wondering about whether the `firstTime` variable name should be something more like `displayFailureMessage` which describes the effects.